flink/v1.17/flink/src/main/java/org/apache/iceberg/flink/FlinkWriteOptions.java

```diff
@@ -64,4 +65,8 @@ private FlinkWriteOptions() {}
 
   public static final ConfigOption<Integer> WRITE_PARALLELISM =
       ConfigOptions.key("write-parallelism").intType().noDefaultValue();
+
+  @Experimental
+  public static final ConfigOption<Long> TABLE_REFRSH_MS =
+      ConfigOptions.key("table-refresh-ms").longType().noDefaultValue();
```

Comment:
this should be a `durationType`

Reply:
I attempted this but it looks like `FlinkConfParser` needs some updates to support duration types. For now, I copied what was being done in `FlinkReadOptions` for a duration type, which makes it a string.

Reply:
it should be `durationType` not `stringType`. we need to add the duration type support in `FlinkConfParser`

Reply:
OK I'll make those updates.

Reply:
I made this change.